src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -33,7 +33,7 @@ using namespace mu::draw;
 using namespace mu::notation;
 
 static constexpr qreal SCROLL_LIMIT_OFF_OFFSET = 0.75;
-static constexpr qreal SCROLL_LIMIT_ON_OFFSET = 0.02;
+static constexpr qreal SCROLL_LIMIT_ON_OFFSET = 0.00;
```

Comment:
I'd just write `0.0` inline, and get rid of this constant. (And while at it, SCROLL_LIMIT_OFF_OFFSET could perhaps use a better name, like `SCROLL_LIMIT_OFF_OVERSCROLL_FACTOR`.)

Reply:
"There are no unnecessary changes" ;-)

One **force-push** for the sake of approval comin' up.